problemsets-mat223/modules/module5.tex

```diff
@@ -103,7 +125,22 @@
 \begin{example}
 	Find the component of $\vec a=\mat{1\\2}$ in the direction of $\vec b=\mat{1\\1}$.
 
-	XXX Finish
+    Let $\Vcomp_{\vec b}\vec a=\mat{x\\y}$ for some unknown $x,y\in\R$. 
+    
+    By definition, $\vec a - \Vcomp_{\vec b}\vec a$ is orthogonal to $\vec b$. That is,
+    \begin{equation}
+    \label{e1}
+        (\vec a-\Vcomp_{\vec b}\vec a)\cdot\vec b=0. \implies \mat{1-x\\2-y}\cdot\mat{1\\1} = 0 \implies x+y=3.
```

Comment:
Quit with the `\implies`. What you have is a string of equalities. Just use `=`.

Reply:
Yeah I quit. T-T